x/staking/types/errors.go

```diff
@@ -50,4 +50,5 @@ var (
 	ErrCommissionLTMinRate             = errors.Register(ModuleName, 40, "commission cannot be less than min rate")
 	ErrUnbondingNotFound               = errors.Register(ModuleName, 41, "unbonding operation not found")
 	ErrUnbondingOnHoldRefCountNegative = errors.Register(ModuleName, 42, "cannot un-hold unbonding operation that is not on hold")
+	ErrInvalidSigner                   = errors.Register(ModuleName, 43, "expected authority account as only signer for proposal message")
```

Comment:
Thank you for the change, I know it is a bit out of scope of this PR but could you update it here too:
- https://github.com/cosmos/cosmos-sdk/blob/629dc63ecdc76687785ede7c7df54dbf7b0afe8a/x/auth/keeper/msg_server.go#L27